<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbsorptionDialog</class>
 <widget class="QDialog" name="AbsorptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Absorption</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxLeft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>66</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxTop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>66</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxBottom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>66</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxRight">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>66</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
